src/SeqCli/Cli/Commands/ApiKey/RemoveCommand.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Seq.Api.Model.Inputs;
+using SeqCli.Cli.Features;
+using SeqCli.Connection;
+
+namespace SeqCli.Cli.Commands.ApiKey
+{
+    [Command("apikey", "remove", "Send a structured log event to the server", Example =
+        "seqcli log -m 'Hello, {Name}!' -p Name=World -p App=Test")]
+    class RemoveCommand : Command
+    {
+        private readonly SeqConnectionFactory _connectionFactory;
+        private readonly ConnectionFeature _connection;
+        private string _title;
+
+        public RemoveCommand(SeqConnectionFactory connectionFactory)
+        {
+            _connectionFactory = connectionFactory;
+            _connection = Enable<ConnectionFeature>();
+            Options.Add(

```

Comment:
Is `id` a worthwhile option/alternative to title? Would help with the ambiguous ones where names overlap.

Reply:
I have added `Id` and put a check in to ensure that they only specify a single value since both could be confusing as to if its removing only when both match or just one.